ex4_ex5/index.js

```diff
@@ -0,0 +1,20 @@
+const prompt = require("prompt-sync")({ sigint: true});
+
+console.log("Insert yout first grade:");
+const firstGrade = prompt(">");
+
+console.log("Insert yout second grade: ");
+const secondGrade = prompt(">");
+
+
+function result() {
+    let sum = (Number(firstGrade) + Number(secondGrade)) / 2;
+
+    if (sum >= 6.0) {
```

Comment:
We usually want to avoid **_Magic Numbers_** ✨. Please consider moving the number `6` to a properly named constant.

```js
const PASSING_GRADE = 6
```